pandas/core/generic.py

```diff
@@ -354,7 +354,7 @@ def _get_axis_number(cls, axis):
                 return cls._AXIS_NUMBERS[axis]
             except KeyError:
                 pass
-        raise ValueError(f"No axis named {axis} for object type {cls}")
+        raise ValueError(f"No axis named {axis} for object type {cls.__name__}")
```

Comment:
can you put ``<{cls.__name__}>``

Reply:
I'm following the format that I was given by @jorisvandenbossche in this pull request: https://github.com/pandas-dev/pandas/pull/32816